### DISCUSSION TITLE
4.1.4 Release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1808] [General]
All devs,

Because #1769 is affecting all new users and many old ones, I'll roll a 4.1.4 fixing release. Current release notes are:
https://github.com/sepinf-inc/IPED/blob/4.1.x/ReleaseNotes.txt

Should we include something else?

**Comment by lfcnassif** [comment]
I probably will include #1809. Any more? #1798?

**Reply by wladimirleite** [reply]
I think it is safe to include both.

**Comment by lfcnassif** [comment]
> I think it is safe to include both.

I agree. Since the versioning semantics will be broken (again), I'll also include #1294, #1815 and maybe #1562 if tests look good, since it was finished months ago...
